src/main/resources/CodyBundle.properties

```diff
@@ -189,3 +189,6 @@ ignore.action-in-ignored-file.title=Cody is Limited
 ignore.status-bar-ignored-file.tooltip=This file has been ignored by an admin.\nAutocomplete, commands, and other Cody features are disabled.
 ignore.sidebar-panel-ignored-file.text=<html>This file has been marked as ignored by an admin, which means commands that rely on its contents cannot be executed.</html>
 ignore.sidebar-panel-ignored-file.learn-more-cta=Learn more
+context-panel.tree.node-ignored.tooltip=Repo ignored by an admin setting
```

Comment:
no strong feelings but personally I prefer the complete name instead of abbreviation so `Repository` instead of `Repo`